docs/source/snippets.rst

```diff
@@ -0,0 +1,37 @@
+XRPL Code Snippets
+===================
+
+Code snippets to demonstrate basic usage of the xrpl-py library.
+
+.. toctree::
+   :maxdepth: 1
+
+Look Up a Transaction on the Ledger
+-------------------------------------
+
+.. literalinclude:: ../../snippets/get_transaction.py
+
+Send a Transaction and See if It Gets Validated
+-----------------------------------------------
+
+.. literalinclude:: ../../snippets/reliable_transaction_submission.py
+
+Change Your Password by Setting a Regular Key
```

Comment:
I don't really like the phrase "password" here - can we leave the title at `Set a Regular Key`?